tools/plugins/com.liferay.ide.upgrade.commands.core/src/com/liferay/ide/upgrade/commands/core/UpgradeCommandsCorePlugin.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.ide.upgrade.commands.core.internal;
+package com.liferay.ide.upgrade.commands.core;
```

Comment:
We must change entry in `META-INF/MANIFEST.MF` that points to `Bundle-Activator`